index.html

```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
               Bootstrap
               <span> <img src="images/image3.png" alt="bootstrap" /></span>
             </li>
-            <li>Ruby</li>
+            <li class="list-sumbua">Ruby</li>
           </ul>
           <button class="seeproject-btn" type="submit">See Project</button>
         </div>
       </section>
+      <form action="https://formspree.io/f/xqkngpry" method="post">
+        <h1>CONTACT US</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          class="input-area"
+          type="text"
+          placeholder="Enter your name"
+          maxlength="30"
+          name="name"
+          required
+        />
+        <input
+          class="input-area mail"
+          type="email"
+          placeholder="Enter your email"
+          name="email"
+          required
+        />
+        <textarea
+          name="textarea"
+          class="text-area"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+        <button class="seeproject-btn" type="button">Submit</button>
```

Comment:
- [ ] Your form currently doesn't work, this is due to this button, for the form to work type must be `type="submit"`. Please fix this issue